index.html

```diff
@@ -8,6 +8,6 @@
     <title>Hello World</title>
```

Comment:
- [ ] Your branch name is **new-branch** which is not very descriptive means it doesn't tell us what are you doing in this branch. You can use something like **feature/setup-hello-microverse** for you branch name.

**Important Note: When you will rename your branch then this pull request will be automatically closed. Then you will need to open a pull request from the renamed branch and you need to make all these changes in that pull request so the next reviewer will approve it :+1:**